NUITizenGallery/res/layout/ScrollViewTest1Page.xaml

```diff
@@ -1,30 +1,32 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<base:View x:Class="NUITizenGallery.ScrollViewTest1Page"
+<comp:ContentPage x:Class="NUITizenGallery.ScrollViewTest1Page"
   xmlns="http://tizen.org/Tizen.NUI/2018/XAML"
   xmlns:base="clr-namespace:Tizen.NUI.BaseComponents;assembly=Tizen.NUI"
   xmlns:comp="clr-namespace:Tizen.NUI.Components;assembly=Tizen.NUI.Components"
   xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+           WidthSpecification="-1"

```

Comment:
use "{Static LayoutParamPolicies.MatchParent}" not -1

Reply:
Changed.